[2025-03-25 12:11:49] [33mwarn[39m: This is a warning log
[2025-03-25 12:11:52] [31merror[39m: This is an error log
[2025-03-25 12:16:37] [33mwarn[39m: This is a warning log
[2025-03-25 12:16:39] [31merror[39m: This is an error log
[2025-04-08 16:06:04] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:39:17] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:39:26] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-08 16:47:53] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
The column `username` does not exist in the current database.
[2025-04-08 16:49:54] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
The column `username` does not exist in the current database.
[2025-04-08 16:51:27] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
The column `username` does not exist in the current database.
[2025-04-08 16:56:23] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:29] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:31] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:40] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:41] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:41] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:42] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:58:58] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:02:00] [33mwarn[39m: Credentials Don't Match
[2025-04-08 17:09:14] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:09:14] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:10:20] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:10:20] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:11:58] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:11:58] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:13:17] [31merror[39m: Service User Not Found: 
[2025-04-08 17:13:57] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:14:30] [31merror[39m: Service User Not Found: 
[2025-04-08 17:29:59] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "password"
    ],
    "message": "Required"
  },
  {
    "expected": "'ADMIN' | 'USER'",
    "received": "undefined",
    "code": "invalid_type",
    "path": [
      "role"
    ],
    "message": "Required"
  }
]
[2025-04-08 17:29:59] [33mwarn[39m: Missing info
[2025-04-08 17:30:56] [31merror[39m: Auth Update User Service:  
Invalid `prisma.user.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:35:22

  32 }
  33 
  34 export async function updateUserDB(id: number, data: User) {
â†’ 35   return prisma.user.update(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-08 17:35:19] [33mwarn[39m: Credentials Don't Match
[2025-04-08 17:36:37] [31merror[39m: Service User Not Found: 
[2025-04-08 17:36:45] [31merror[39m: Auth Delete User Service:  
Invalid `prisma.user.delete()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:11:28

   8 }
   9 
  10 export async function deleteUserDB(id: number) {
â†’ 11   return prisma.user.delete(
An operation failed because it depends on one or more records that were required but not found. Record to delete does not exist.
